src/helpers/passive-fn.ts

```diff
@@ -163,16 +164,65 @@ export const passive = (params: PassiveDef): AnySoupElement[] => {
 
   if (pw === undefined) throw new Error("could not infer pad width")
   if (ph === undefined) throw new Error("could not infer pad width")
+  // if (params.w && params.p && params.pw) {
+
+  const PcbSilkscreenPath: PcbSilkscreenPath[] = [
```

Comment:
Use camelCase for variable names. PascalCase is conventionally used for interfaces, types or classes